starter-code/src/viking.js

```diff
@@ -1,11 +1,81 @@
 // Soldier
-function Soldier() {}
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;    
+    this.attack = function() {
```

Comment:
Try to be consistent. Except for this one, you are declaring the methods for Soldier, Viking and Saxon outside the constructor (recommended). Put this one outside too.